templates/digital-asset-template/frontend/utils/assetsUploader.ts

```diff
@@ -46,18 +56,17 @@ export const uploadCollectionData = async (
     return;
   }
 
-  // check collection thumbnail image exists
-  let mediaExt: string;
-  const collectionCover = files.find((file) =>
-    VALID_MEDIA_EXTENSIONS.some((ext) => {
-      if (file.name.endsWith(`collection.${ext}`)) {
-        mediaExt = ext;
-        return true;
-      } else {
-        return false;
-      }
-    })
+  const collectionCover = collectionFiles.find((file) =>
+    VALID_MEDIA_EXTENSIONS.some((ext) => file.name.endsWith(`.${ext}`))
   );
+  if (!collectionCover) {
+    alert(
+      "Collection cover not found, please make sure you include the collection image file"
+    );
+    return;
+  }
+
+  const mediaExt = collectionCover?.name.split(".").pop();
```

Comment:
we use mediaExt later to find the nft image files, right? and we assume it would have the same extension as the collection cover one 
https://github.com/aptos-labs/create-aptos-dapp/blob/c7a23bb2733812360ea237bfd84cd6d1f2da7ce2/templates/digital-asset-template/frontend/utils/assetsUploader.ts#L93

Reply:
correct